content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -2,50 +2,91 @@
 Ratings
 =======
 
-Allow customers to rate their experience with your helpdesk teams to strengthen your credibility and
-gain their trust. Reviews can also influence a customer’s decision and open space for feedback that
-can help you improve the quality of your services.
+Asking customers to rate their experience with :guilabel:`Helpdesk` support provides the opportunity
+to gauge team performance, and track customer satisfaction. Ratings can published to the portal,
+providing customers with a statistical overview of the team's performance.
 
 Set up
 ======
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable *Ratings on
-tickets*. The feature automatically adds a default email template on the non-folded *closing
-stage(s)* of that team.
+To enable :guilabel:`Customer Ratings`, go to :menuselection:`Helpdesk --> Configuration --> Teams`.
+Select a team, scroll to :guilabel:`Performance`, and select :guilabel:`Customer Ratings`.
 
-.. image:: ratings/ratings_on_tickets.png
+.. image:: ratings/ratings_enable.png
    :align: center
    :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
          in Odoo Helpdesk
 
-To edit the email template and the stage(s) set as the closing ones, go to the Kanban view of your
-helpdesk team and click on *Settings*, then on *Edit Stage*.
+To automatically request ratings from customers once a ticket has closed, an email template must
+be added to the appropriate stage. Click the :guilabel:`Set an Email Template on Stages` button.
+Select a :guilabel:`Stage`.
 
-.. image:: ratings/edit_stage.png
+.. important::
+   Customers should only be asked to rate tickets once their issue has been resolved and their ticket
+   is closed. Therefore, a *Ratings Request* email should only be added to a stage that is
+   :guilabel:`Folded in Kanban`, as tickets in a folded stage are considered closed.
+
+.. image:: ratings/ratings_template_onstage.png
    :align: center
    :alt: Overview of a helpdesk team kanban view emphasizing the menu edit stage in Odoo Helpdesk
 
-Now, once a ticket reaches the stage(s) designated as the *Closing Stage*, an email is sent to the
-customer.
+Select an email template, or create a new one. Click :guilabel:`Save & Close`. Now, once a ticket
+reaches the folded stage an email is sent to the customer requesting a rating.
```

Comment:
Should "folded stage" be `:guilabel:` or italicized?

Reply:
I think italicized, since it should be emphasized (it is a pretty significant change from 15) but it isn't a direct reference in the UI.